scripts/restructure-ideas-image-urls.js

```diff
@@ -0,0 +1,38 @@
+const db = require('../src/db');
+
+async function updateImageUrls() {
+
+    try {
```

Comment:
I think this try catch is too general. It catches both the db not connected (according to the error). But it will also catch any other error in this code or when the new item can not be saved correctly.

I think you have to split them up somehow.

Reply:
Yes, the `findAll()` and `save()` commands now have there own try catch